roles/common/tasks/ssl.yml

```diff
@@ -7,6 +7,14 @@
             dest=/usr/local/share/ca-certificates/{{ endpoints.main }}.crt
   notify: refresh CAs
 
+- name: Install any additional CA certificates
+  copy:
+    content: "{{ item.content | default(omit) }}"
+    dest: "/usr/local/share/ca-certificates/{{ item.name }}.crt"
+    src: "{{ item.src | default(omit) }}"
+  with_items: "{{ cacerts | default([]) }}"
```

Comment:
ok, the reason i had it outside of that is global cacerts is something i would expect to be common to a whole deployment - installed on every machine. 
In my case i am using hostvars to provide the individual hosts cert and key with the ssl hash. If i add the cacerts to the ssl hash in defaults these will be overridden by the ssl hash in hostvars and i would need to add the common CA list to every host. right? 
Happy to change it just explaining the rationale.